worklist/views_helper_functions/store_articles.py

```diff
@@ -52,7 +51,7 @@ def store_psid_articles(worklist_object, psid, created_by):
     # Fetching articles from petscan using psid
     results = fetch_articles_from_petscan(psid)
 
-    if results['success'] is False:
+    if results is False or results['success'] is False:
```

Comment:
This is a pretty cumbersome return value, especially since we're not even using results['message'].

How about making it so that fetch_articles_from_petscan just returns an empty list on error?

Reply:
Done.